### ISSUE TITLE
Support for CurrentPhase1-3 with insights

### ISSUE THREAD

**Issue by eranpeso** [Bas-vE/nl.amazeme.slimmelezer#2]
Hello,

I looked through the code and saw that "CurrentPhase1", "CurrentPhase2" & "CurrentPhase3" is commented out in device.js and not used in the driver. I would really appreciate support for CurrentPhase-data from SlimmeLezer in Homey with possibility to have this historical values available through Homey insights.
Do you have any plans to implement this?

**Comment by Bas-vE** [comment]
So the thing is, there are a lot more data i can gather, but I don't want my users to be overwhelmed with all the insight data this app generates. Also when I look at my current phase of the slimmelezer webinterface (just browse to the slimmelezer ip address)

I see no change in the data. the data always looks like this on my side

![image](https://github.com/Bas-vE/nl.amazeme.slimmelezer/assets/16207111/078f151c-9efb-4ac2-a28a-42c7b10a71f1)
 
If your slimmelezer gives you more data than this, I will consider on adding it to the app.

**Comment by eranpeso** [comment]
I totally agree, my first reflection using the app for Homey is that i wanted to remove "Electricity produced", "Power consumed T1", "Power consumed T2", "Power produced T1", "Power produced T1", "Power produced total", "Gas total" since none of these are giving me values or are not relevant to me (no production).
In Sweden (my home & meter) I get data for the different phases and this data is quite important as I can see how the electrical loads are balanced between the different phases. Se screenshot from my Slimmelezer+ below:

![image](https://github.com/Bas-vE/nl.amazeme.slimmelezer/assets/116366004/61891d83-8082-44b5-9072-71b238e22b65)

The ideal solution is if it would be possible in the device settings for Homey what data should be collected. But I assume this is not possible?

**Comment by Bas-vE** [comment]
Thats very interesting to see. I may need to look into the documentation to see if its possible to add functionality to enable/disable insight data.

**Comment by Melectro1** [comment]
I also live in Sweden, and I have a strong need to monitor the current on each individual phase. The reason is that I want to limit the charging current of my EV charger to avoid overloading and potentially tripping the main fuse.

Therefore, it would be very helpful if you could add some flow cards such as:
"When" current phase 1 is greater than xx Amps
"When" current phase 2 is greater than xx Amps
"When" current phase 3 is greater than xx Amps

Additionally, I assume that the current values can be read and used as input arguments in, for example, a script or similar logic.

<img width="931" height="698" alt="Image" src="https://github.com/user-attachments/assets/f3d242eb-c46b-4a7f-925c-b8135e328963" />
